javascript-light-bulb/script.js

```diff
@@ -0,0 +1,30 @@
+var off = 0;
+var $body = document.querySelector('.background-color-off');
+var $lightBulb = document.querySelector('.light-bulb-off');
+
+$lightBulb.addEventListener('click', function () {
+  off++;
+  if (off % 2 === 0) {
```

Comment:
Instead of using a counter, I would use a Boolean value that you can toggle to `true` or `false`